eng/common/tools.sh

```diff
@@ -353,9 +353,9 @@ function InitializeBuildTool {
 function GetNuGetPackageCachePath {
   if [[ -z ${NUGET_PACKAGES:-} ]]; then
     if [[ "$use_global_nuget_cache" == true ]]; then
-      export NUGET_PACKAGES="$HOME/.nuget/packages"
```

Comment:
I actually don't know if this is supposed to happen in our repo or arcade 🤷

Reply:
in Arcade. Anything in `common` will be overwritten with the next arcade bump I'm afraid.

Reply:
ah, I'll go do a pr over there then